package.json

```diff
@@ -203,7 +203,6 @@
     "immortal-db": "^1.1.0",
     "immutable": "4.0.0-rc.12",
     "js-cookie": "^3.0.0",
-    "lodash": "^4.17.21",
```

Comment:
👍

Reply:
Although by lodash recommendation we should be using the full library instead the split packages. This way of using lodash will disappear in v5

https://lodash.com/per-method-packages